libtcmu_log.c

```diff
@@ -692,10 +697,31 @@ int tcmu_setup_log(char *log_dir)
 	return -ENOMEM;
 }
 
+static bool is_path_same(const char* path1, const char* path2)
+{
+	struct stat st1 = {0,};
+	struct stat st2 = {0,};
+
+	if (!path1 || !path2)
+		return false;
+
+	if ((stat(path1, &st1) == -1) || (stat(path2, &st2) == -1)) {
+		return false;
+	}
+
+	return st1.st_dev == st2.st_dev && st1.st_ino == st2.st_ino;
+}
+
 int tcmu_resetup_log_file(char *log_dir)
 {
 	int ret;
 
+	if (is_path_same(tcmu_log_dir, log_dir)) {
+		tcmu_dbg("Skipping resetting log_dir, as it matches with "
+		         "current log dir path: %s.\n", log_dir);
```

Comment:
Usually the log format won't be wrapped and split into 2 or more lines.
We may make it shorter like:
```

tcmu_dbg("Nothing changes for log dir: %s, skip it.\n",
                  log_dir)
```

Reply:
@lxbsz 
actually it prints in one single line. there is no '\n' in between the lines ?
Or are you saying lets just make it short ?

Reply:
Yeah, it is. I meant in the code.